web/app/scripts/custom-reports/custom-reports-modal-controller.js

```diff
@@ -66,7 +66,9 @@
         }
 
         function assembleParams() {
-            return QueryBuilder.assembleParams(0, true, true).then(
+            // TODO: if there's a boundary selected, should that filter be applied when
+            // they haven't selected a geographical aggregation? Currently it's never applied.
```

Comment:
It's applied everywhere else, right? It would be confusing to have different behavior here.